src/Raven.Server/ServerWide/ClusterStateMachine.cs

```diff
@@ -3237,11 +3239,8 @@ public override async Task OnSnapshotInstalledAsync(long lastIncludedIndex, Serv
             }
             token.ThrowIfCancellationRequested();
 
-            // reload license can send a notification which will open a write tx
-            serverStore.LicenseManager.ReloadLicense();
-            await serverStore.LicenseManager.PutMyNodeInfoAsync();
```

Comment:
We need this also on a full snapshot

Reply:
which is exactly this line:
```
if (fullSnapshot)
          await OnValueChanges(lastIncludedIndex, nameof(PutLicenseCommand));
```

take a look here:
https://github.com/ravendb/ravendb/blob/aa68edc645e3951ff432f630ca54f43c91df86f3/src/Raven.Server/ServerWide/ServerStore.cs#L1048